packages/statemanager/src/simpleStateManager.ts

```diff
@@ -125,6 +118,8 @@ export class SimpleStateManager implements EVMStateManagerInterface {
     this.topStorageStack().set(`${address.toString()}_${bytesToHex(key)}`, value)
   }
 
+  async clearStorage(): Promise<void> {}
```

Comment:
Why is there no implementation of this one?

Reply:
There was non before, so there is no status quo change here, but we might add (just would not want to do it now, have this other PR on the plate as well). I tested just re-setting the internal storage dict, but *this* actually then made tests fail in VM/EVM so I removed again.